ChemCheck/upload/static_app/JS/aceeditor.js

```diff
@@ -12,4 +20,4 @@ $(document).ready(function(){
         editor.setValue($("#file-content").text());
         editor.setOptions({
         maxLines: 2000
-        });
\ No newline at end of file
+        });}
```

Comment:
You should add a blank newline at the end of this file. Also make sure that `db.sqlite3` is added to `.gitignore`

Reply:
Thanks Bryan. I just changed them.